l10n_it_fatturapa_out/wizard/efattura.py

```diff
@@ -194,13 +194,17 @@ def get_all_taxes(record):
             SdI expects a summary for every tax mentioned in the invoice,
             even those with price_total == 0.
             """
+
+            def _key(tax_id):
+                return "{}".format(tax_id.id)
+
             out_computed = {}
             # existing tax lines
             tax_ids = record.line_ids.filtered(lambda line: line.tax_line_id)
             for tax_id in tax_ids:
                 tax_line_id = tax_id.tax_line_id
                 aliquota = format_numbers(tax_line_id.amount)
-                key = "{}_{}".format(aliquota, tax_line_id.kind_id.code)
+                key = _key(tax_line_id)
```

Comment:
perché non semplicemente `str(tax_line_id.id)`?

Reply:
Si ma si era deciso di fare una funzione per calcolare la chiave, visto che viene usata in più punti, cosi se la si cambia ancora è più difficile dimenticarsi di farlo dappertutto.

Reply:
Ok d'accordo, e come mai non usare `str` in quella funzione ?

Reply:
Beh perché prima aveva un formato diverso, poi è stata ridotta al solo id in corso d'opera. Per altro, manco serve sia una stringa.